core/app/components/FormBuilder/FormElement.tsx

```diff
@@ -63,22 +67,24 @@ export const FormElement = ({ element, index, isEditing, isDisabled }: FormEleme
 				removeElement(index);
 			}}
 		>
-			<Trash size={24} className="text-neutral-400 hover:text-red-500" />
+			<Trash size={24} className="!pointer-events-auto text-neutral-400 hover:text-red-500" />
```

Comment:
@tefkah do you recall why we are setting `pointer-events: none` in the button component? it breaks the `:hover` pseudo-class so i ended up overriding it here.

https://github.com/pubpub/platform/commit/5eebab0b94054dd522f47e4a3081ca7f7590eb9d#diff-54e976c9e25a1c11aa59c088a36d31acd884abaeb450882b0e0b0e5076a83508R10